course_catalog/tasks.py

```diff
@@ -34,8 +34,27 @@ def get_mitx_data():
 
 
 @app.task(acks_late=True)
+def get_ocw_courses_with_retry(
+    *, course_prefixes, blocklist, force_overwrite, upload_to_s3
+):
+    """
+    Task to sync a batch of OCW courses. We want acks_late=True for tasks launched
+    with the webhook so that there will be a retry if the task fails or the worker is killed
+    """
+    sync_ocw_courses(
+        course_prefixes=course_prefixes,
```

Comment:
Since the webhook is always for just one course_prefix, you could change this task name to `get_ocw_course_with_retry`, the input parameter name to `course_prefix`, and the code to:
```python
sync_ocw_courses(
    course_prefixes=[course_prefix]
    ...
```
and also in `course_catalog.views.py`, change the input parameter there from a list to a single prefix:
```
                get_ocw_course_with_retry.apply_async(
                    countdown=settings.OCW_WEBHOOK_DELAY,
                    kwargs={
                        "course_prefix": prefix,
```